src/components/TopBanner.jsx

```diff
@@ -67,17 +67,17 @@ const PageDesc = styled(Box)(({ theme }) => ({
   },
 }))
 
-const TopBanner = ({ bannerImg, pageTitle, pageDesc }) => {
+const TopBanner = ({ bannerImg, pageTitle, pageDesc, isDonate }) => {
```

Comment:
I understand what you're trying to do here with the `isDonate` prop, but I think we should try to not add a page-specific prop like this. We want to be able to use the `TopBanner` component on all the pages, so it wouldn't make sense to pass in a prop for each page—technically we could do it, it would work, but the logic would get really complicated.

Instead there are two options: If the pages are really this different, instead of adding a bunch of branching logic like this, it's just better to copy into a new component where maybe 90% of the code is the same. Then you can pull out any shared logic into smaller sub-components.

However, in this case, do these pages really need to be this different? Most of the differences here seem to be color and spacing tweaks. I would just simplify things (e.g. use the same background color on both). The current designs are just a rough guide, it's not worth adding in this much complexity to be pixel perfect. If there are any issues you run into that truly need to be different for the donate page, let me know and I can help